client/src/pages/tasks/Show.js

```diff
@@ -293,6 +302,26 @@ const TaskShow = ({ pageDispatchers }) => {
                       }
                     />
                   )}
+                  {Settings.fields.task.childrenTasks && task.childrenTasks.length > 0 && (
+                    <TaskChildrenTasks
+                      dictProps={Settings.fields.task.childrenTasks}
```

Comment:
Change these three lines:
```javascript
-                  {Settings.fields.task.childrenTasks && task.childrenTasks.length > 0 && (
-                    <TaskChildrenTasks
-                      dictProps={Settings.fields.task.childrenTasks}
+                  {Settings.fields.task.childrenTasks && task.childrenTasks?.length > 0 && (
+                    <Field
+                      label={Settings.fields.task.childrenTasks}
```
since it does not match the dictProps of DictionaryField (it's not an input field).